pyne/partisn.py

```diff
@@ -472,36 +454,19 @@ def _check_fine_mesh_total(block01):
         if key in ['it', 'jt', 'kt']:
             total += block01[key]
     
-    if total >= 7:
-        # no warning necessary
-        return False
-    else:
-        # warn the user
-        return True
+    if total < 7:
```

Comment:
Just out of curiosity @elliottbiondo, did you actually run into this problem where if you had less than 7 total it wouldn't work? It seems like a very odd number but @makeclean said his wasn't working for < 7.